### PR TITLE
Reptilian sounds

### PR DESCRIPTION
**Pull request by ghost**
# Description
Add hiss and one additional lizard sound.
---

# Changelog

:cl:
- add: Reptilian hiss and another very neat sound

### CODE REVIEW

**Review by ShadesMars — CHANGES_REQUESTED** [review]
Test fails are valid.

File: Resources/Prototypes/Floof/Voice/speech_emote.yml
Comment (Memeji) [inline Resources/Prototypes/Floof/Voice/speech_emote.yml:1]: Should add the ReptilianHiss to the speech_emote too, I believe.

**Review by Memeji — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Floof/Voice/speech_emote.yml
Comment (Fansana) [reply Resources/Prototypes/Floof/Voice/speech_emote.yml:1]: It is, Resources/Prototypes/Voice/speech_emote_sounds.yml line 95

**Comment by Fansana** [comment]
Seems like the test cases have nothing to do with the PR anymore, please review. @FoxxoTrystan @Memeji

File: Resources/Audio/Floof/Voice/Reptilian/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- files: ["yip.ogg"]
+  license: "CC-BY-3.0""
+  copyright: "Original sound added by https://github.com/NopemanMcHalt"
+  source: "https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/blob/master/modular_splurt/sound/voice/yip.ogg"
+
+- files: ["hiss.ogg"]
+  license: "CC-BY-3.0""
```
Comment (FoxxoTrystan) [inline Resources/Audio/Floof/Voice/Reptilian/attributions.yml:7]:
```suggestion
  license: "CC-BY-3.0"
  copyright: "Original sound added by https://github.com/NopemanMcHalt"
  source: "https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/blob/master/modular_splurt/sound/voice/yip.ogg"

- files: ["hiss.ogg"]
  license: "CC-BY-3.0"
```

**Review by ShadesMars — CHANGES_REQUESTED** [review]

**Review by ShadesMars — APPROVED** [review]

**Pull request merged**